site/zenodo_rdm/migrator/README.md

```diff
@@ -65,6 +70,18 @@ invenio vocabularies import -v names -f ./app_data/vocabularies-future.yaml  # z
 python -m zenodo_rdm.migrator /path/to/streams.yaml
 ```
 
+```bash
+# Manual COPY ...
+export DB_URI=${SQLALCHEMY_DATABASE_URI}
```

Comment:
curiosity: this is not done programmatically because?

Reply:
this is needed for the manual copy of the csv via the psql only